server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/MailboxSetDeletePerformer.scala

```diff
@@ -120,6 +124,16 @@ class MailboxSetDeletePerformer @Inject()(mailboxManager: MailboxManager,
                   .`then`())
             }
           }))
+      .doOnSuccess(_ => auditTrail(mailboxSession, id))
+
+  private def auditTrail(mailboxSession: MailboxSession, id: MailboxId): Unit =
+    ReactorUtils.logAsMono(() => AuditTrail.entry
+      .username(() => mailboxSession.getUser.asString())
+      .protocol("JMAP")
+      .action("Mailbox/set delete")
+      .parameters(() => ImmutableMap.of("loggedInUser", mailboxSession.getLoggedInUser.toScala.map(_.asString()).getOrElse(""),
+        "mailboxId", id.serialize()))
```

Comment:
IMO we should audit trail by mailboxIds (which mailboxIds for Mailbox/set destroy that client really requests to the server). Doing this would make the impression that the client always requests Mailbox/set destroy one by one mailbox.

See audit trail for Email/set destroy.